erroranalysis/erroranalysis/_internal/constants.py

```diff
@@ -11,3 +14,56 @@
 METHOD = 'method'
 METHOD_EXCLUDES = 'excludes'
 METHOD_INCLUDES = 'includes'
+
+
+class ModelTask(str, Enum):
+    """Provide model task constants.
+
+    Can be 'classification', 'regression' or 'unknown'.
+
+    Note: Keeping sentence case constants (Classification, Regression)
+    for backwards compatibility, please use ALL_UPPER_CASE instead.
+    """
+
+    CLASSIFICATION = 'classification'
+    REGRESSION = 'regression'
+    UNKNOWN = 'unknown'
+    Classification = 'classification'
+    Regression = 'regression'
+
+
+class Metrics(str, Enum):
+    """Provide the supported error analysis metrics.
+
+    The regression metrics are 'mean_absolute_error',
+    'mean_squared_error', 'median_absolute_error',
+    and 'r2_score'.  The classification metrics are
+    'f1_score', 'precision_score', 'recall_score' and
+    'error_rate'.
+    """
+    MEAN_ABSOLUTE_ERROR = 'mean_absolute_error'
+    MEAN_SQUARED_ERROR = 'mean_squared_error'
+    MEDIAN_ABSOLUTE_ERROR = 'median_absolute_error'
+    R2_SCORE = 'r2_score'
+    F1_SCORE = 'f1_score'
+    PRECISION_SCORE = 'precision_score'
+    RECALL_SCORE = 'recall_score'
+    ERROR_RATE = 'error_rate'
+
+
+metric_to_display_name = {
+    Metrics.MEAN_ABSOLUTE_ERROR: 'Mean absolute error',
+    Metrics.MEAN_SQUARED_ERROR: 'Mean squared error',
+    Metrics.MEDIAN_ABSOLUTE_ERROR: 'Median absolute error',
+    Metrics.R2_SCORE: 'R2 score',
```

Comment:
This is R squared, isn't it? Just wondering if this displays with this string.

Reply:
yes it's R squared, but I can't display the square so I just made it R2